tests/plugins/richcombo/richcombo.js

```diff
@@ -89,6 +93,62 @@ bender.test( {
 		} );
 	},
 
+	// (#4493)
+	'test richcombo aria-labelledby attribute points to label element of the combobox': function() {
+		var editor = this.editor,
+			combo = editor.ui.get( 'custom_combo' ),
+			comboBtnSelector = '#cke_' + combo.id,
+			comboBtn = CKEDITOR.document.findOne( comboBtnSelector + ' .cke_combo_button' ),
+			comboBtnLabelElementId = comboBtnSelector.substr( 1 ) + '_label';
+
+		combo.createPanel( editor );
+
+		assert.areEqual( comboBtnLabelElementId, comboBtn.getAttribute( 'aria-labelledby' ), 'Aria-labelledby attribute should point to element with the combo label.' );
+	},
+
+	// (#4493)
+	'test richcombo initial label': function() {
```

Comment:
Could we improve this test case name, so that it will express what we are expecting? I mean something like: `test created richcombo label contains initial label value`